modules/gentemplates/openapi-generator/src/main/java/com/reprezen/genflow/openapi/generator/OagCodegenGenTemplateBase.java

```diff
@@ -64,6 +64,32 @@ public void configure() throws GenerationException {
 				"Each property should be a JSON object with a name/value pair for each property.",
 				"Example: for '-Dmodels -Dapis=User,Pets' use the following:", "value:", "  models: ''",
 				"  apis: Users,Pets"));
+		define(parameter().named(LANGUAGE_SPECIFIC_PRIMITIVES).optional().withDescription(
+				"Specifies additional language specific primitive types.",
```

Comment:
"language-specific" should be hyphenated. 
Also, maybe this would be a good place to clarify that these are not really "primitive" types, or at least not reinforce that misnomer. Suggested edit is to replace the first line with these two lines:
> "Specifies types that are provided by the API implementation, and so should not be generated.", 
> "Type names here should be unqualified. The fully-qualified name should be in IMPORT_MAPPINGS."